bibtex-completion.el

```diff
@@ -846,17 +866,24 @@ find a PDF file."
   (cl-remove-duplicates entry
                         :test (lambda (x y) (string= (s-downcase x) (s-downcase y)))
                         :key 'car :from-end t))
-
 
-(defun bibtex-completion-format-entry (entry width)
+(defun bibtex-completion-format-entry (entry width &optional alt-display-formats)
   "Formats a BibTeX ENTRY for display in results list.
-WIDTH is the width of the results list.  The display format is
-governed by the variable `bibtex-completion-display-formats'."
+WIDTH is the width of the results list. The display format is
+governed by the variable `bibtex-completion-display-formats', or
+by ALT-DISPLAY-FORMATS if present."
   (let* ((format
-          (or (assoc-string (bibtex-completion-get-value "=type=" entry)
-                            bibtex-completion-display-formats-internal
-                            'case-fold)
-              (assoc t bibtex-completion-display-formats-internal)))
+          ;; TODO: need to activate
+          ;; bibtex-completion-display-formats-suffix-internal
+          (if alt-display-formats
+              (or (assoc-string (bibtex-completion-get-value "=type=" entry)
+                                bibtex-completion-display-formats-suffix-internal
+                                'case-fold)
+                  (assoc t bibtex-completion-display-formats-suffix-internal))
+              (or (assoc-string (bibtex-completion-get-value "=type=" entry)
+                                bibtex-completion-display-formats-internal
+                                'case-fold)
+                  (assoc t bibtex-completion-display-formats-internal))))
```

Comment:
I believe this core piece of code is correct, but I have to test it more. Specifically, it works for me with default (no suffix defined), but I need to test if the suffix generation actually works as expected.